src/registry.cpp

```diff
@@ -149,13 +151,38 @@ uint32_t jitc_registry_id(const void *ptr) {
 
 uint32_t jitc_registry_id_bound(JitBackend backend, const char *domain) {
     Registry &r = registry;
+    if (!domain) {
+        uint32_t i = 0;
```

Comment:
```suggestion
        uint32_t n = 0;
```
since it's a count and not an index.